kuma/settings/common.py

```diff
@@ -1665,6 +1665,34 @@ def parse_iframe_url(url):
 # Google Analytics Tracking Account Number (0 to disable)
 GOOGLE_ANALYTICS_ACCOUNT = config('GOOGLE_ANALYTICS_ACCOUNT', default=None)
 
+# When HTTP posting event to Google Analytics this is the combined connect
+# and read timeout.
+GOOGLE_ANALYTICS_TRACKING_TIMEOUT = config(
+    'GOOGLE_ANALYTICS_TRACKING_TIMEOUT',
+    cast=float,
+    default=2.0)
+# The only reason you'd want to override this is for local development where
+# you might want to substitute the events tracking URL to a local dev server.
+# https://developers.google.com/analytics/devguides/collection/protocol/v1/reference
+GOOGLE_ANALYTICS_TRACKING_URL = config(
+    'GOOGLE_ANALYTICS_TRACKING_URL',
+    default='https://www.google-analytics.com/collect'
+)
+GOOGLE_ANALYTICS_TRACKING_URL = config(
+    'GOOGLE_ANALYTICS_TRACKING_URL',
+    default='https://www.google-analytics.com/collect'
+)
```

Comment:
This code should be removed, since it's a duplicate of the code immediately above.